liquidprompt

```diff
@@ -401,6 +407,25 @@ lp_activate() {
         # the feature in .liquidpromptrc)
         (( ${LP_ENABLE_RUNTIME-0} || ${LP_ENABLE_RUNTIME_BELL-0} || ${LP_ENABLE_TITLE_COMMAND-0} )) && trap - DEBUG
 
+        # Disable previous hooks as options that set them
+        # may have changed
+        # (This is "remove the specified value(s) from the array", which bash does not make easy.)
+        new_precmd_functions=()
+        for value in "${precmd_functions[@]}"; do
```

Comment:
This (and other such `for` statements) should also use the `${array[@]+"${array[@]}"}` syntax.
```suggestion
        for value in ${precmd_functions[@]+"${precmd_functions[@]}"}; do
```

Reply:
Done in 058f082